### ISSUE TITLE
No named parameter after update 3.10

### ISSUE THREAD

**Issue by fakhravari** [SachinGanesh/screenshot#139]
Cache/hosted/pub.dev/screenshot-1.3.0/lib/screenshot.dart:160:7: Error: No named parameter with the name 'window'.
      window: ui.window,
      ^^^^^^
/C:/flutter_windows_3.7.3-stable/flutter/packages/flutter/lib/src/rendering/view.dart:68:3: Context: Found this candidate, but the arguments don't match.
  RenderView({
  ^^^^^^^^^^
Target kernel_snapshot failed: Exception

FAILURE: Build failed with an exception.

**Comment by luisrodas-digital** [comment]
This happend with new version of Flutter 3.10.0

**Comment by ctnadovich** [comment]
I see this issue too. Was just about to report it, but I see @fakhravari  beat me to it. It took me about 20 minutes to use RepaintBoundary(key: myKey, child: myStuffToScreenshot) instead, eliminating need for this broken library.  

Then this 
```
RenderRepaintBoundary boundary = previewContainer.currentContext!
          .findRenderObject() as RenderRepaintBoundary;
      ui.Image image = await boundary.toImage();
      final directory = (await getApplicationDocumentsDirectory()).path;
      var byteData = await image.toByteData(format: ui.ImageByteFormat.png);
      if (byteData == null) {
        throw const FormatException("Failed converting image to byte data.");
      }
      Uint8List pngBytes = byteData.buffer.asUint8List();
      // print(pngBytes);
      File imgFile = File('$directory/$filename');
      imgFile.writeAsBytes(pngBytes);

```

**Comment by armandojimenez** [comment]
Any way to fix this? I want to test master flutter branch

**Comment by marcellocamara** [comment]
Unfortunately the owner is taking too long to fix it
Setting this will work:

```
  screenshot:
    git:
      url: https://github.com/SachinGanesh/screenshot.git
      ref: master
```

**Comment by h3x4d3c1m4l** [comment]
Fix seems to be published: https://pub.dev/packages/screenshot

**Comment by ritheshSalyan** [comment]
Published version `2.0.0` with support for flutter `3.10`
